packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -5,10 +5,11 @@
 import 'dart:async';
 import 'dart:collection';
 import 'dart:developer';
-import 'dart:ui' show AppLifecycleState;
+import 'dart:ui' show AppLifecycleState, Canvas, Image, Picture, PictureRecorder;
```

Comment:
the scheduler shouldn't know about these things. They have nothing to do with scheduling.

Reply:
Removed changes to scheduler.